Updated the about page

gavindizon opened the pull request:
Update the about page which includes the about.html and info.json.

gavindizon committed:
Updated the about page

Update the about page including the which includes the about.html info.json.

mjarenyap commented on layouts/partials/pages/about.html:
```diff
@@ -46,20 +46,12 @@ <h2 class="margin-bottom-16">Monthly Meetups</h2>
 				<h2 class="margin-bottom-16">Design Conferences</h2>
 				<p class="margin-bottom-24">{{ .Site.Data.info.conferences }}</p>
-				<button class="outline">Visit our conference website</button>
+				<button class="outline" onclick="window.open('https://roadshow.uxph.org')">Visit UXPH Roadshow</button>
```
Just wrap this under an `a` tag instead of using `onclick` function. Make sure to set the `target` to `blank`.

gavindizon committed:
Revised about.html according to the comment

Changed the onlick button function to an 'a' wrapper to redirect to the UXPH roadshow
